src/main/include/Robot.h

```diff
@@ -6,10 +6,10 @@
 /*----------------------------------------------------------------------------*/
 
 #pragma once
-
+#include <frc/DriverStation.h>
 #include <frc/TimedRobot.h>
 #include <frc2/command/Command.h>
-
+#include <subsystems/DooHickey.h>
```

Comment:
Use `"` for our includes

```suggestion
#include "subsystems/DooHickey.h"
```